package.json

```diff
@@ -24,7 +24,7 @@
     "lint:js": "eslint '{src,test}/**/*.{js,jsx}'",
     "testcafe:drive": "node testcafe/runner-drive.js",
     "testcafe:photos": "node testcafe/runner-photos.js",
-    "test": "env USE_REACT=true cozy-scripts test --config=jest.config.json",
+    "test": "env NODE_ENV='test' cozy-scripts test --config=jest.config.json",
```

Comment:
Did it in the commit message! But in case you're genuinely curious about this and don't want to check the commit message, it's because of https://github.com/cozy/cozy-ui/pull/1365

Reply:
YOU FORCE PUSHED, right 🤣 Sorry for the noise